packages/editor-ui/src/stores/assistant.store.ts

```diff
@@ -587,6 +592,12 @@ export const useAssistantStore = defineStore(STORES.ASSISTANT, () => {
 						text: chatMessage.text,
 						quickReplyType: chatMessage.quickReplyType,
 						context: userContext,
+						workflowContext: {
+							currentWorkflow: workflowDataStale.value ? workflowsStore.workflow : undefined,
+							executionData: workflowExecutionDataStale.value
+								? workflowsStore.workflowExecutionData?.data?.resultData
```

Comment:
why do we send nothing if data is stale? Seems like this will happen often no?

Reply:
This is just to optimize it a bit. The latest sent version is still available in the AI service session data if you are worried about it. 
This should prevent sending the data necessarily as workflows can be quite large.